_includes/specials/taproot/en/11-vaults-with-taproot.md

```diff
@@ -1,7 +1,10 @@
-*By [Antoine Poinsot][darosior], Revault developer* <!-- FIXME: darosior or Antoine, GitHub/Twitter/something else?  -->
+*By [Antoine Poinsot][darosior], Revault developer*
 
-Bitcoin [vaults][topic vaults] are contracts playing on the finality of a payment. Numerous such protocols have been
-proposed (single or multi-party, with or without covenants) so we'll focus on what they have in
+Bitcoin [vaults][topic vaults] are contracts that can reveal to a
+user that one or more of their signing keys has been obtained by an
+attacker, and which may allow the user to prevent the attacker from
```

Comment:
Not really? It's completely different to my previous definition and does not apply to all vaults proposals.

A common pattern to all proposals is the usage of a "staging" transaction (the unvault) so that a payment is not final until both the Unvault and the (timelocked) Spend transaction have been confirmed.

Reply:
I've replaced this with:

"Bitcoin vaults are a type of contract that require two sequential transactions for a user to spend money from their wallet. Numerous such protocols have been proposed..."

Are you happy with that using that as the introductory sentence?